src/ticks/debugger.c

```diff
@@ -1038,7 +1040,26 @@ static int cmd_quit(int argc, char **argv)
     exit(0);
 }
 
+static int cmd_restore(int argc, char **argv)
+{
+    int address;
+    if ( argc == 3 ) {
+        address = parse_address(argv[2]);
+    } else {
+        address = symbol_resolve("__head");
+        if (address == -1) {
+            printf("Warning: could not resolve starting address and no address is provided.\n");
+            return 0;
+        }
+    }
 
+    if (bk.restore(argv[1], address))
```

Comment:
Is it worth taking an argument on the command line that is the binary and restore defaults to using that?

Reply:
I thought about that but z88dk-gdb doesn't take binary as an argument. We might add one just so this command couild use it. Ticks doesn't have this command implemented at all.

Reply:
Okay, I'll pick this up when I add the ticks command.